### ISSUE TITLE
[BUG] Creating NVTabular model for Triton Inference with different input/output column names

### ISSUE THREAD

**Issue by bschifferer** [NVIDIA/NVTabular#630]
**Describe the bug**
Currently we support only if input_names=output_names. But what if the NVTabular workflow renames columns AND/OR uses more complicated workflows, adding new features to it.

The issue is in model.py when we initialize the workflow. We iterate over self.workflow.input_dtypes and try to extract the output dtypes from the config file. If the input column name is not in the output column name (e.g. due to renaming), the initialization will fail

```
class TritonPythonModel:
    def initialize(self, args):
        workflow_path = os.path.join(
            args["model_repository"], str(args["model_version"]), "workflow"
        )
        self.workflow = nvtabular.Workflow.load(workflow_path)
        self.model_config = json.loads(args["model_config"])

        self.output_dtypes = dict()
        for name in self.workflow.input_dtypes:
            conf = get_output_config_by_name(self.model_config, name)
            self.output_dtypes[name] = triton_string_to_numpy(conf["data_type"])
```

**Steps/Code to reproduce bug**

**Expected behavior**
An easy fix seems to be to extract the output column names from another variable (e.g. below)

```
        self.output_dtypes = dict()
        output_names = [x['name'] for x in self.model_config['output']]
        for name in output_names:
            conf = get_output_config_by_name(self.model_config, name)
            self.output_dtypes[name] = triton_string_to_numpy(conf["data_type"])
```

**Comment by benfred** [comment]
This should be fixed by https://github.com/NVIDIA/NVTabular/pull/614/files#diff-bb609d3863256a67880448f3a08c1ee9bec0cb9160ab7570fc8454f6b3af6455R67

**Comment by oyilmaz-nvidia** [comment]
@benfred Sounds good. Will remove this part from my bugfix branch then.

**Comment by benfred** [comment]
Fixed by #614
